src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,383 @@
+	 *
+	 * @param ReaderThemes $reader_themes ReaderThemes instance.
+	 */
+	public function __construct( ReaderThemes $reader_themes ) {
+		$this->reader_themes = $reader_themes;
+	}
+
+	/**
+	 * Gets the value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Key for the option.
+	 *
+	 * [--format=<format>]
+	 * : Get value in a particular format.
+	 * ---
+	 * default: var_export
+	 * options:
+	 *   - var_export
+	 *   - json
+	 *   - yaml
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # Get option.
+	 * $ wp amp option get theme_support
+	 * standard
+	 *
+	 * # Get option in JSON format.
+	 * $ wp amp option get theme_support --format=json
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function get( $args, $assoc_args ) {
+		list( $option_name ) = $args;
+
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		$options = $this->get_options();
+
+		if ( $options instanceof WP_Error ) {
+			/* translators: %s: error message */
+			WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+		}
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Could not get %s option. Does it exist?', 'amp' ), $option_name ) );
+		}
+
+		WP_CLI::print_value( $options[ $option_name ], $assoc_args );
+	}
+
+	/**
+	 * Updates an option value.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to update.
+	 *
+	 * <value>
+	 * : The new value.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # Update plugin option.
+	 * $ wp amp option update theme_support reader
+	 * Success: Updated theme_support option.
+	 *
+	 * @alias set
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function update( $args, $assoc_args ) {
+		list( $option_name, $option_value ) = $args;
+
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		$options = $this->get_options();
+
+		if ( $options instanceof WP_Error ) {
+			/* translators: %s: error message */
+			WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+		}
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Could not update %s option. Does it exist?', 'amp' ), $option_name ) );
+		}
```

Comment:
Why do we need to get all the options? This doesn't seem to be used.

Reply:
This is used to determine whether the user is trying to update the option which exists in `amp-options`. I added it before allowed options notice so that the user can learn that he is trying to update a non-existent option.

If we remove this guard then the user will see:
```php
if ( ! in_array( $option_name, self::ALLOWED_OPTIONS, true ) ) {
    /* translators: %s: option name */
    WP_CLI::error( sprintf( __( 'You are not allowed to update %s option via the CLI.', 'amp' ), $option_name ) );
}
```
for every attempt to update a wrong option.

Reply:
I think we can update the message here to something like this:
```diff
if ( ! isset( $options[ $option_name ] ) ) {
    /* translators: %s: option name */
-  WP_CLI::error( sprintf( __( 'Could not update %s option. Does it exist?', 'amp' ), $option_name ) );
+   WP_CLI::error( sprintf( __( 'You are trying to update %s option, but it does not exist.', 'amp' ), $option_name ) );
}
```

Reply:
I see. Yes, it can be removed now that we have `ALLOWED_OPTIONS`. And the update to the message looks good!

Reply:
IMO `ALLOWED_OPTIONS` notice will be a bit unclear when the user will try to update the wrong option i.e running `wp amp option update sandboxing_enable true` will result in `You are not allowed to update sandboxing_enable option via the CLI.` while with options check guard the message will `You are trying to update sandboxing_enable option, but it does not exist.`

Do you still want me to remove highlighted logic?

Reply:
How about the message say something like:

```The option %1$s is not among the following options that can currently be managed via CLI: %1$2.```

Where the former is the provided option, and the latter is a comma-separated list of `ALLOWED_OPTIONS`

Reply:
Perfect. Will update it 👍🏼